<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacherCreateFacultative</class>
 <widget class="QDialog" name="teacherCreateFacultative">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>152</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Create your facultative</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>152</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name of your facultative:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit">
         <property name="placeholderText">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Facultative start date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="startDateEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Facultative end date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="endDateEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Choose day of your facultative:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dayBox">
         <item>
          <property name="text">
           <string>Monday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tuesday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wednesdey</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Thursday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Friday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Saturday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sunday</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="picture">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>:/Downloads/calendar-symbol-svgrepo-com.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Type of Lesson</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="laboratoryBox">
         <item>
          <property name="text">
           <string>Lection</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Laboratory work</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Practice</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
